### PR TITLE
fix "Errors retrieving from CouchDB"

### PR DESCRIPTION
**Pull request by qibobo**
[#118007371]

### CODE REVIEW

File: server/src/main/java/org/cloudfoundry/autoscaler/data/couchdb/dao/base/TypedCouchDbRepositorySupport.java [hunk 1]
```diff
@@ -134,6 +134,24 @@ public T get (String id) {
...
     }
     
+    public T tryGet (String id) {
+		String[] input = beforeConnection("GET",  new String[]{id});
+		T returnvalue = null;
+		try{
+    	 returnvalue = super.get(id);
+		} catch (Exception e){
+			String msg = e.getMessage();
+			if(msg.indexOf("nothing found on db path") >= 0){
```
Comment (fraenkel) [inline server/src/main/java/org/cloudfoundry/autoscaler/data/couchdb/dao/base/TypedCouchDbRepositorySupport.java]: Why isn't this following the same pattern as below?

Comment (qibobo) [inline server/src/main/java/org/cloudfoundry/autoscaler/data/couchdb/dao/base/TypedCouchDbRepositorySupport.java]: Just updated the code. Please have a check. Thank you!

**Comment by fraenkel** [comment]
@qibobo Please format your code before submitting.

**Comment by fraenkel** [comment]
Merged

**Pull request closed without merging**